### PR TITLE
Contributing MAT I/O Library

### PR DESCRIPTION
**Pull request by tloizou**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/m/MATIO/MATIO-1.5.2-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+description = """matio is an C library for reading and writing Matlab MAT files."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%s-%s.zip' % (name.lower(), version)]
+
+
+dependencies = [('zlib','1.2.7'),
+]
```
Comment (boegel) [inline easybuild/easyconfigs/m/MATIO/MATIO-1.5.2-goolf-1.4.10.eb]: collapse to a single line please

File: easybuild/easyconfigs/m/MATIO/MATIO-1.5.2-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+name = 'MATIO'
+version = '1.5.2'
+
+homepage = 'http://sourceforge.net/projects/matio/'
+description = """matio is an C library for reading and writing Matlab MAT files."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%s-%s.zip' % (name.lower(), version)]
```
Comment (boegel) [inline easybuild/easyconfigs/m/MATIO/MATIO-1.5.2-goolf-1.4.10.eb]: pleaseuse `SOURCELOWER_ZIP`

File: easybuild/easyconfigs/m/MATIO/MATIO-1.5.2-goolf-1.4.10.eb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/m/MATIO/MATIO-1.5.2-goolf-1.4.10.eb]: drop double newline

**Comment by boegel** [comment]
missing `easyblock = 'ConfigureMake'` ?

**Comment by tloizou** [comment]
@boegel: I have corrected the config. Ready for testing.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/903c873a4895715aa57b for a full test report.

**Comment by boegel** [comment]
Looking good, going in, thanks @tloizou!

**Pull request merged**